week-3/C-array-every/exercise.js

```diff
@@ -5,14 +5,15 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = group.every(checkForStudents);
 
-if (groupIsOnlyStudents) {
-  console.log("The group contains only students");
-} else {
-  console.log("The group does not contain only students");
+function checkForStudents(groupIsOnlyStudents) {
+  if (groupIsOnlyStudents.includes(students)) {
+    console.log("The group contains only students");
```

Comment:
I don't think this is quite right. Remember that the `.every()` method expects you to return a boolean, so that it find determine if every element in the array matches the condition in your predicate function.

The exercise is a little bit confusing as it doesn't run the `console.log(groupIsOnlyStudents)` for you, but you can add that after calling `.every()`